src/main/resources/application.properties

```diff
@@ -1 +1,12 @@
+jdbc.driver=com.mysql.jdbc.Driver
+jdbc.url=jdbc:mysql://localhost:3306/dev_db
+jdbc.username=root
+jdbc.password=root
 
+
+url.prefix = GET/weather-conditions" + "\\?" + "location=
```

Comment:
Remember to remove the `GET` prefix as this I belive this is not what they intended.

Reply:
yes, I removed it